src/coreclr/jit/emit.h

```diff
@@ -3857,6 +3857,15 @@ emitAttr emitter::emitGetMemOpSize(instrDesc* id) const
             return EA_32BYTE;
         }
 
+        case INS_vcvttss2usi64:
+        {
+            if (defaultSize == 8)
+            {
+                return EA_4BYTE;
+            }
+            return defaultSize;
```

Comment:
Is this right? I'd expect this should be `EA_4BYTE` always.

I would expect we're either passing down `EA_4BYTE` already, as its generated for regular "scalar" code -or- we're passing down a vector size and this can be 16

Reply:
We can change this to return ```EA_4BYTE``` always. But we do need this handling for ```vcvttss2usi64``` since we are going from 4 bytes to 8 bytes. This function is specifically used to handle instructions which have a different input and output sizes. Without this code, we would return ```EA_*BYTE``` and we would read a ```qword``` from instead of a ```dword```. 

I returned ```defaultSize``` in all other cases to be a little cautious if there are other cases. But now that we know there are none, we can always return ```EA_4BYTE```

Reply:
> Without this code, we would return EA_*BYTE and we would read a qword from instead of a dword.

Right, but it is only ever valid to read 4-bytes, since that is what the instruction does. So we should make sure to track it accurately here.